src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,42 @@
+package seedu.addressbook.data.tag;
+
+import java.util.*;
+import seedu.addressbook.data.person.Person;
+
+public class Tagging {
+
+    public static final String CONTACT_DELETED = "-";
+    public static final String CONTACT_ADDED= "+";
+    public static final String SPACE = " ";
+
+    private boolean isContactDeleted;
+    private Person person;
+    private Tag tag;
+
+    // Constructor 
+    public Tagging(Person person, Tag tag, boolean isContactDeleted){
```

Comment:
`isTagDeleted`?
A boolean variable works here, but an `enum` may be more appropriate because the two values we need are `added` and `deleted`, not `true` and `false`.